queries/buildings.jinja2

```diff
@@ -57,10 +58,11 @@ SELECT
     NULL AS area,
     osm_id AS __id__,
     NULL AS "building:part",
+    "address" AS kind,
```

Comment:
@nvkelso do we want to update the [transform](https://github.com/mapzen/TileStache/blob/7ca4b5e2f87e6218522a981315e00c379d80bdc8/TileStache/Goodies/VecTiles/transform.py#L156-L164) to leave the `kind` alone if it's already set?

Reading through it, it should work as is because all the other fields would be null anyway, but we might want to be more explicit in the transform just for clarity.